docs/source/algorithms/PowderDiffILLDetEffCorr-v1.rst

```diff
@@ -9,84 +9,196 @@
+
+NumberOfIterations
 ------------------
 
-:ref:`PowderDiffILLReduction <algm-PowderDiffILLReduction>` performs the data reduction.
+This is used for **D2B** only.
+For **D20** there is no need for iterations, since a single shot derivation is already convergent; that is, the residual calibration factors are identical to unity.
+
+This specifies how many times the calibration has to be derived (see Step 4 above for **D2B** method):
+
+  - 1 by default: The calibration will be derived only once (single-shot) and no iteration will be performed. Typically this gives reasonably good result already.
 
-Usage
------
+  - User specified positive integer: Iterations will be performed as many times as requested. It is not advised to iterate too much, since after local convergence it may start to diverge again; hence there is a hard limit of 10.
 
-**Example - PowderDiffILLDetEffCorr**
+  - 0 stands for auto: Iterations will be run automatically until the termination criteria is satisfied. Termination criteria is:
+
+      .. math:: \chi^2/NdoF = \frac{\sum_{i,j}(c_{ij} - 1)ˆ2}{N_{tubes} * N_{pixels_per_tube}} < t
+
+      where :math:`c_{ij}` is the residual calibration factor for tube *i* and pixel *j*, :math:`t` is the threshold defined in :ref:`Instrument Parameter File (IPF)<InstrumentParameterFile>` as *chi2_ndof*.
+
+      The top and bottom parts of the tubes are excluded from this calculation. How many pixels exactly are excluded is again defined in :ref:`IPF <InstrumentParameterFile>` as *pixels_to_trim*.
+
+      Currently, for **D2B**, *pixels_to_trim=28* and *t=1%*. With this settings iterations typically terminate after the first one, i. e. one run and one iteration give results already convergent within 1%.
+
+      This has to be interpreted as: the residual calibration is close enough to unity, so further iterations will not change the calibration much.
+
+      However, this criterion does not prevent from divergence in all the cases.
+      It can happen that for a given pixel the residual calibration factor (albeit close to unity) is always on the same side (i.e. always above 1 or below 1); this will cause the absolute calibration to gradually diverge with iterations.
+      Anyways, the method implemented does not provide enough precision to resolve residual calibration better than in the percent range.
+      Hence, care must be taken when using the iterations.
+      It is not recommended to use more than 2 iterations.
+
+Limitations
+-----------
+
+For **D2B** it is assumed that the tubes and pixels pass through the exact same positions during the scan.
+That is, the tubes have to be aligned vertically and horizontally and the gap between each pair of neighboring tubes must be integer multiple of the scan step.
+
+D20 Workflow
+------------
+
+.. diagram:: PowderDiffILLDetEffCorr-v1_D20_wkflw.dot
+
+**Example - D20**
```

Comment:
Is the example data available in Mantid's usage example data package? If so, it should be mentioned here as well. There is some sphinx directive to add a standard phrase and link here.